docs/04-architecture/turbopack.mdx

```diff
@@ -22,7 +22,145 @@ Turbopack can be used in Next.js in both the `pages` and `app` directories for f
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    turbo: {
+      rules: {
+        // Option format
+        '*.md': [
+          {
+            loader: '@mdx-js/loader',
+            options: {
+              format: 'md',
+            },
+          },
+        ],
+        // Option-less format
+        '*.mdx': ['@mdx-js/loader'],
+      },
+    },
+  },
+}
+```
+
+> **Good to know**:
+>
+> Note: Prior to Next.js version 13.4.4, `experimental.turbo.rules` was named
+> `experimental.turbo.loaders` and only accepted file extensions like `.mdx`
+> instead of `*.mdx`.
+
+If you need to pass something like the result of importing an external package as a loader option, it's possible to wrap the webpack loader with your own, specifying options there. **This is an interim solution and should not be necessary in the future.**
+
+This loader wraps `@mdx-js/loader` and configures the `rehypePrism` rehype plugin:
+
+```js filename="my-mdx-loader.js"
+const mdxLoader = require('@mdx-js/loader')
+const rehypePrism = require('@mapbox/rehype-prism')
+
+module.exports = function (code) {
+  const prevGetOptions = this.getOptions.bind(this)
+  this.getOptions = function getOptions(...args) {
+    return {
+      ...prevGetOptions(...args),
+      rehypePlugins: [rehypePrism],
+    }
+  }
+
+  mdxLoader.call(this, code)
+}
+```
+
+Then, configure Next.js to load the wrapper loader:
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    turbo: {
+      rules: {
+        '*.mdx': ['./my-mdx-loader'],
+      },
+    },
+  },
+}
+```
+
+### Supported loaders
+
+The following loaders have been tested to work with Turbopack's webpack loader implementation:
+
+- [`babel-loader`](https://www.npmjs.com/package/babel-loader)
+- [`@mdx-js/loader`](https://www.npmjs.com/package/@mdx-js/loader)
+- [`@svgr/webpack`](https://www.npmjs.com/package/@svgr/webpack)
+- [`svg-inline-loader`](https://www.npmjs.com/package/svg-inline-loader)
+- [`yaml-loader`](https://www.npmjs.com/package/yaml-loader)
+- [`string-replace-loader`](https://www.npmjs.com/package/string-replace-loader)
+- [`raw-loader`](https://www.npmjs.com/package/raw-loader)
+
+## Resolve aliases
+
+Turbopack can be configured to modify module resolution through aliases, similar to webpack's [`resolve.alias`](https://webpack.js.org/configuration/resolve/#resolvealias) configuration:
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    turbo: {
+      resolveAlias: {
+        underscore: 'lodash',
+        mocha: { browser: 'mocha/browser-entry.js' },
+      },
+    },
+  },
+}
+```
+
+This aliases imports of the `underscore` package to the `lodash` package so that `import underscore from 'underscore'` will load the `lodash` module instead of `underscore`.
```

Comment:
Nit, for clarity. 

```suggestion
For example, this aliases the `underscore` package to the `lodash` package so that `import underscore from 'underscore'` will load the `lodash` module instead of `underscore`.
```